src/styles/header.scss

```diff
@@ -0,0 +1,57 @@
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-inline-start: 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  padding: 0;
+  margin: 0 50px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  height: $header-height;
```

Comment:
The variable `$header-height` should be defined in your variables file or elsewhere in your project to ensure it is recognized here.